xcode/Subconscious/Shared/Services/Noosphere/SphereFS.swift

```diff
@@ -264,6 +264,74 @@ public final class SphereFS: SphereProtocol {
         })
     }
     
+    // TODO: Do these belong on SphereFS?
```

Comment:
Similar question RE: is this the right place for these methods, they make sense as part of `SphereFS` today but as we know that abstraction is going away in the medium term.

Reply:
If I understand @cdata, address book belongs to a sphere, so I believe this is the right place to put these.